dev/validate_bom__config.py

```diff
@@ -1003,7 +1003,31 @@ class SecurityConfigurator(Configurator):
   pass
 
 
+class SpinnakerConfigurator(object):
+  """Controls spinnaker-local overrides."""
+
+  def init_argument_parser(self, parser):
+    """Implements interface."""
+    pass
+
+  def validate_options(self, options):
+    """Implements interface."""
+    pass
+
+  def add_config(self, options, script):
+    """Implements interface."""
+    script.append('mkdir -p  ~/.hal/default/profiles')
+    script.append('echo "management.security.enabled: false"'
```

Comment:
Guard this with a flag. Add the flag to #1011. You can default it off for now and I'll turn it on with my PR later.